cmd/server/assets/mobileapps/_app.html

```diff
@@ -2,41 +2,75 @@
 
 {{$app := .app}}
 
+<style>
```

Comment:
should this be in application.css?

Reply:
Meh, if we end up using it across multiple files we can move it over.